src/simplewallet/simplewallet.cpp

```diff
@@ -3245,6 +3245,7 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
   {
     try
     {
+      m_trusted_daemon = false;
```

Comment:
Doesn't the line above

```c++
    if (!m_trusted_daemon)
```
already imply this?

Reply:
No, that line tests whether m_trusted_demon has a value (it's a boost::optional<bool>), whereas the assignment sets a value.

Reply:
Ah, right.